kong/plugins/moesif/schema.lua

```diff
@@ -16,6 +17,7 @@ return {
     batch_size = {default = 200, type = "number"},
     disable_transaction_id = {default = false, type = "boolean"},
     debug = {default = false, type = "boolean"},
+    enable_skip_gzip_payload_decompression = {default = true, type = "boolean"},
```

Comment:
may be rename it as skip_gzip_payload_decompression?

Reply:
also default it to be false. We wanted it to be true for ware2go.. once he tests everything..and if works..we can ask him to override this to true

Reply:
Agree, but can we call it disable_gzip_payload_decompression rather than skip?